### ISSUE TITLE
Trying to send a MQTT notification throws an error

### ISSUE THREAD

**Issue by MericFeyz** [ScorpioBroker/ScorpioBroker#387]
Following the [tutorial](https://scorpio.readthedocs.io/en/latest/mqtt.html) in the docs, I have tried to setup sending of notifications through MQTT but got an error instead. Following is the error I got: 
```
2023-04-28 12:27:14,771 DEBUG [io.qua.oid.run.OidcUtils] (vert.x-eventloop-thread-1) No claim exists at the path 'groups' at the path segment 'groups'
2023-04-28 12:27:14,783 DEBUG [eu.nec.ngs.sub.mes.SubscriptionMessagingBase] (pool-6-thread-7) Append got called: urn:ngsi-ld:Vehicle:A135
2023-04-28 12:27:14,831 ERROR [eu.nec.ngs.com.sub.NotificationHandlerMQTT] (pool-6-thread-7) Exception ::: java.lang.IllegalArgumentException: no NetworkModule installed for scheme "mqtt" of URI "mqtt://host.k3d.internal:1883"
        at org.eclipse.paho.mqttv5.client.internal.NetworkModuleService.validateURI(NetworkModuleService.java:72)
        at org.eclipse.paho.mqttv5.client.MqttAsyncClient.<init>(MqttAsyncClient.java:573)
        at org.eclipse.paho.mqttv5.client.MqttAsyncClient.<init>(MqttAsyncClient.java:439)
        at org.eclipse.paho.mqttv5.client.MqttClient.<init>(MqttClient.java:280)
        at org.eclipse.paho.mqttv5.client.MqttClient.<init>(MqttClient.java:172)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.NotificationHandlerMQTT.getClient(NotificationHandlerMQTT.java:114)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.NotificationHandlerMQTT.sendReply(NotificationHandlerMQTT.java:35)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseNotificationHandler.notify(BaseNotificationHandler.java:71)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseSubscriptionService.sendNotification(BaseSubscriptionService.java:551)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseSubscriptionService.checkSubscriptions(BaseSubscriptionService.java:537)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseSubscriptionService.checkSubscriptionsWithDelta(BaseSubscriptionService.java:764)
        at eu.neclab.ngsildbroker.subscriptionmanager.messaging.SubscriptionMessagingBase.handleBaseRequestEntity(SubscriptionMessagingBase.java:52)
        at eu.neclab.ngsildbroker.subscriptionmanager.messaging.SubscriptionMessagingBase$1.run(SubscriptionMessagingBase.java:32)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)

2023-04-28 12:27:15,807 DEBUG [io.qua.oid.run.OidcUtils] (vert.x-eventloop-thread-0) No claim exists at the path 'groups' at the path segment 'groups'
2023-04-28 12:27:15,818 DEBUG [eu.nec.ngs.sub.mes.SubscriptionMessagingBase] (pool-6-thread-8) Append got called: urn:ngsi-ld:Vehicle:A135
2023-04-28 12:27:15,820 ERROR [eu.nec.ngs.com.sub.NotificationHandlerMQTT] (pool-6-thread-8) Exception ::: java.lang.IllegalArgumentException: no NetworkModule installed for scheme "mqtt" of URI "mqtt://host.k3d.internal:1883"
        at org.eclipse.paho.mqttv5.client.internal.NetworkModuleService.validateURI(NetworkModuleService.java:72)
        at org.eclipse.paho.mqttv5.client.MqttAsyncClient.<init>(MqttAsyncClient.java:573)
        at org.eclipse.paho.mqttv5.client.MqttAsyncClient.<init>(MqttAsyncClient.java:439)
        at org.eclipse.paho.mqttv5.client.MqttClient.<init>(MqttClient.java:280)
        at org.eclipse.paho.mqttv5.client.MqttClient.<init>(MqttClient.java:172)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.NotificationHandlerMQTT.getClient(NotificationHandlerMQTT.java:114)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.NotificationHandlerMQTT.sendReply(NotificationHandlerMQTT.java:35)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseNotificationHandler.notify(BaseNotificationHandler.java:71)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseSubscriptionService.sendNotification(BaseSubscriptionService.java:551)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseSubscriptionService.checkSubscriptions(BaseSubscriptionService.java:537)
        at eu.neclab.ngsildbroker.commons.subscriptionbase.BaseSubscriptionService.checkSubscriptionsWithDelta(BaseSubscriptionService.java:764)
        at eu.neclab.ngsildbroker.subscriptionmanager.messaging.SubscriptionMessagingBase.handleBaseRequestEntity(SubscriptionMessagingBase.java:52)
        at eu.neclab.ngsildbroker.subscriptionmanager.messaging.SubscriptionMessagingBase$1.run(SubscriptionMessagingBase.java:32)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)

```
I am running the 3.0.8 Version of Scorpio All in One Runner with the in-memory Profile and the Quarkus app is running inside of a k3d cluster, therefore I am using host.k3d.internal as the host. For the record,  the subscriber client I am using is mosquitto_sub and I can use the mosquitto_pub client inside the Scorpio pod to reach the subscriber client on the host machine with no issues.

**Comment by amitrnec** [comment]
Hi,
We are working on this issue. we will fix asap.

Thanks.

**Comment by amitrnec** [comment]
Hi,
We have fixed this issue. Fix code will be in next release.

Thanks.

**Comment by ScorpioBroker** [comment]
fixed in 4.0.3
